### DISCUSSION TITLE
How can I use linux CLI version on single terminal and sub/pub?

### DISCUSSION THREAD

**Discussion by Flithor** [emqx/MQTTX#1504] [Q&A] [answered]
I use command `mqttx sub -t 'topic'` to subscribe the topic and cannot use publish command in interactive?

**Comment by Red-Asuka** [comment, marked as answer]
The sub and pub commands cannot be used simultaneously on a single terminal. You can use terminal multiplexers such as screen or tmux to create multiple windows and panels, and switch between these windows and panels.

**Comment by Red-Asuka** [comment]
Some SSH clients, such as PuTTY or MobaXterm, provide options to open multiple SSH sessions in the same application window. Each session has its own label or window, and you can easily switch between them.

**Comment by Flithor** [comment]
OK, I see
